### ISSUE TITLE
quick commit not working?

### ISSUE THREAD

**Issue by kevin-coelho** [divmain/GitSavvy#996]
- Sublime Text version number: 3.1.1
- Git version number: 2.17.1
- OS type and version: macOS high sierra, 10.13.6
- Console error output, 
```
Traceback (most recent call last):
   File "/Users/kcoelho/Library/Application Support/Sublime Text 3/Installed Packages/GitSavvy.sublime-package/core/commands/quick_commit.py", line 23, in <lambda>

   File "/Users/kcoelho/Library/Application Support/Sublime Text 3/Installed Packages/GitSavvy.sublime-package/core/commands/quick_commit.py", line 27, in on_done

   AttributeError: 'GsQuickCommitCommand' object has no attribute 'view'
```
- A description of the problem.

> Git quick commit does not work

- Steps to reproduce, if possible

> run the quick commit command.

- If the bug is caused by a failing command, [include a debug log](https://github.com/divmain/GitSavvy/blob/master/docs/debug.md#providing-a-debug-log). 

> I tried to follow the instructions but the log showed only `[]`.

**Comment by randy3k** [comment]
Thanks for the report. We'll fix it asap.

**Comment by randy3k** [comment]
@kevin-coelho it has been fixed and 2.17.5 will soon be available on Package Control.
